# Copyright 2018 NoMagic Sp. z o.o.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

Header header
bool standard_0
bool standard_1
bool standard_2
bool standard_3
bool standard_4
bool standard_5
bool standard_6
bool standard_7

bool configurable_0
bool configurable_1
bool configurable_2
bool configurable_3
bool configurable_4
bool configurable_5
bool configurable_6
bool configurable_7

bool tool_0
bool tool_1
